index.html

```diff
@@ -601,6 +755,63 @@
             }.bind(layer.id));
         }
 
+        var f3 = gui.addFolder('WFS Features');
+        var fPoint = f3.addFolder('Point');
+        var fLine = f3.addFolder('Line');
+        text['testWfsPoint'] = true;
+        fPoint.add(text, 'testWfsPoint').onChange(function(newValue) {
+            itowns.viewer.setLayerVisibility('testWfsPoint', newValue);
+        });
+
+        controlText['retailType'] = 'circle';
```

Comment:
It is not supposed to change the type of details you have, it is suppose to change the area of detailed features (the detailed features are represented as boxes and the others as circles). The center of the 'circle' and of the 'box' are initially close to the center of Lyon.
When 'circle' is chosen the detailed area is a circle and the center of this area can be changed by pressing D and clicking right on another part of the map.
When 'box' is chosen, the detailed area is a rectangle and it can be changed by pressing D and clicking right twice on the map.
When 'custom' is chosen, the detailed area depends on the tile level and change according to the levelSwitchTab which is instantiated in the NodeManager class.